src/main/java/connectify/model/Note.java

```diff
@@ -0,0 +1,43 @@
+package connectify.model;
+
+/**
+ * Represents a Note in the address book.
+ */
+public class Note {
+    private String content;
+
+    /**
+     * Constructor for Note.
+     */
+    public Note() {
+    }
+
+    /**
+     * Constructor for Note.
+     * @param content Content of note
+     */
+    public Note(String content) {
+        this.content = content;
+    }
+
+    public String getContent() {
+        return content;
+    }
+
+    @Override
+    public String toString() {
+        return content;
+    }
```

Comment:
Do you think it would be good to standardise? Having Note as a abstract class and having a PersonNote subclass?

Reply:
Thanks for the suggestion! In my opinion, I decided to use a common Note class because there is no difference in the use case for both the Person and Company. What do you think?

Reply:
After further discussion, I have made the changes to make "Note" an abstract class for future extensible purposes! Do review my work once again! Thank you!

Reply:
Thanks for the changes!